addons/im_livechat/models/mail_message_reaction.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models
+from odoo.addons.mail.tools.discuss import Store
+
+
+class MailMessageReaction(models.Model):
+    _name = "mail.message.reaction"
+    _inherit = ["mail.message.reaction"]
+
+    def _reacting_persona_to_store(self, store: Store):
+        channel_reactions_with_partner = self.filtered(
+            lambda r: r.partner_id and r.message_id.model == "discuss.channel"
+        )
+        channel_by_message = channel_reactions_with_partner.message_id._record_by_message()
+        livechat_partner_reactions = channel_reactions_with_partner.filtered(
+            lambda r: channel_by_message.get(r.message_id, self.env["discuss.channel"]).channel_type
+            == "livechat"
+        )
+        super(MailMessageReaction, self - livechat_partner_reactions)._reacting_persona_to_store(
+            store
+        )
```

Comment:
do we really need the reacting prefix on the reaction model? can we gain a line here if we remove it?